src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 101;
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
ColorSupplier should be class-level variable

Reply:
Why should the 'colorSupplier' be a class-level variable while the 'ball' is located within the method?